src/utils/tracking/trackEvent.ts

```diff
@@ -0,0 +1,18 @@
+export const trackEvent = (
```

Comment:
I'd suggest using `quepasa` for this instead 🙂

Reply:
sure! is it ready tho? we kinda need to get this out as we're shipping traffic to the wrong place now 🙈

Reply:
You can do this for a while, but you should consider replacing it very soon (`quepasa` should be stable enough this week)

Reply:
amaze! ping me when its ready? :D